### DISCUSSION TITLE
Can Aggrid cells render Button blcock

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1141] [Questions and Help]
Hi, @SamTolmay 
I now use lowdefy's `aggrid`. I want to realize the render delete `Button block` in the last cell of each form. Can this be realized?

**Comment by Gervwyk** [comment]
Hi @xiaoxiji1234 

Unfortunately not at the moment, but you can use the `onCellClick` event to process an action when a cell is clicked, or skip, if it is not the right cell. 

Rendering blocks inside of an aggrid cell would make a lot of sense, we've also had to implement workarounds like `onCellClick` to achieve such functionality. We'll revisit this in the future, to create a block that could do this.

**Reply by qin-zhuopu** [reply]
Hi @Gervwyk 

its been over 2 years, any updates? is this feature available now.
